app.js

```diff
@@ -27,11 +18,22 @@ window.AppData = new Proxy(defaultData, {
         window.console.error('Unable to parse l10n strings: ', ex);
       }
     } else obj[prop] = value;
```

Comment:
Hmm, I don't think I'm very concerned about this type of bug, we only send data down in launch-video, i think It would be excessive to add these checks/validation